src/components/Admin/resources/competition/semester/SemesterList.tsx

```diff
@@ -11,35 +11,50 @@ import {
 } from 'react-admin'
 
 import {DateTimeField} from '@/components/Admin/custom/DateTimeField'
+import {CompetitionFilterSection} from '@/components/Admin/custom/list-filtering/CompetitionFilterSection'
+import {FilterSidebar} from '@/components/Admin/custom/list-filtering/FilterSidebar'
 import {SeasonCodeField} from '@/components/Admin/custom/SeasonCodeField'
 
 export const SemesterList: FC = () => (
-  <List>
+  <List aside={<SemesterListFilters />}>
     <Datagrid>
-      <ReferenceField source="competition" reference="competition/competition" link={false} />
+      <ReferenceField source="competition" reference="competition/competition" link={false} sortable={false} />
       <NumberField source="year" />
-      <SeasonCodeField source="season_code" />
-      <TextField source="school_year" />
+      <SeasonCodeField source="season_code" sortable={false} />
+      <TextField source="school_year" sortable={false} />
       <DateTimeField source="start" />
       <DateTimeField source="end" />
-      <BooleanField source="complete" />
-      <TextField source="additional_name" />
-      <NumberField source="registration_link" />
+      <BooleanField source="complete" sortable={false} />
+      <TextField source="additional_name" sortable={false} />
+      <NumberField source="registration_link" sortable={false} />
       <FunctionField<RaRecord>
         source="series_set"
         label="Series count"
         render={(record) => record && <span>{record['series_set'].length}</span>}
+        sortable={false}
       />
       <FunctionField<RaRecord>
         source="publication_set"
         label="Publication count"
         render={(record) => record && <span>{record['publication_set'].length}</span>}
+        sortable={false}
       />
       <FunctionField<RaRecord>
         source="late_tags"
         label="Late tags count"
         render={(record) => record && <span>{record['late_tags'].length}</span>}
+        sortable={false}
       />
     </Datagrid>
   </List>
 )
+
+const SemesterListFilters: FC = () => (
+  <FilterSidebar>
+    <CompetitionFilterSection />
```

Comment:
tu by sme to mali iba podla seminara filtrovat, nie polda competition

alebo to aspon prefiltrovat len na tie hodnoty co davaju zmysel nakolko competitions je plna prdel (to iste pre `competition/series` a `competition/problems`

Reply:
ez 
<img width="195" alt="image" src="https://github.com/user-attachments/assets/92697db4-7173-40ce-8d67-995c24d0f87d" />